### PR TITLE
[zh]Resync concepts files after zh language renaming(concepts-5)

### PR DESCRIPTION
**Pull request by howieyuen**
related: https://github.com/kubernetes/website/issues/34221

### CODE REVIEW

File: content/zh-cn/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -53,8 +52,7 @@ Resource quotas work like this:
   the `LimitRanger` admission controller to force defaults for pods that make no compute resource requirements.
   See the [walkthrough](/docs/tasks/administer-cluster/quota-memory-cpu-namespace/) for an example of how to avoid this problem.
 -->
-- 不同的团队可以在不同的命名空间下工作，目前这是非约束性的，在未来的版本中可能会通过
-  ACL (Access Control List 访问控制列表) 来实现强制性约束。
+- 不同的团队可以在不同的命名空间下工作。这可以通过 [RBAC](/zh/docs/reference/access-authn-authz/rbac/) 强制执行
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/resource-quotas.md:55]:
```suggestion
- 不同的团队可以在不同的命名空间下工作。这可以通过 [RBAC](/zh/docs/reference/access-authn-authz/rbac/) 强制执行。
```

File: content/zh-cn/docs/concepts/cluster-administration/addons.md [hunk 2]
```diff
@@ -84,8 +84,8 @@ Add-ons 扩展了 Kubernetes 的功能。
   CaaS / PaaS 平台（例如关键容器服务（PKS）和 OpenShift）之间的集成。
 * [Nuage](https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.1.1-1/docs/kubernetes-1-installation.rst)
   是一个 SDN 平台，可在 Kubernetes Pods 和非 Kubernetes 环境之间提供基于策略的联网，并具有可视化和安全监控。
-* Romana 是一个 pod 网络的第三层解决方案，并支持
-  [NetworkPolicy API](/zh/docs/concepts/services-networking/network-policies/)。
+* [Romana]((https://github.com/romana)) 是一个 Pod 网络的第三层解决方案，并支持
+  [NetworkPolicy](/zh/docs/concepts/services-networking/network-policies/) API。
   Kubeadm add-on 安装细节可以在[这里](https://github.com/romana/romana/tree/master/containerize)找到。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/cluster-administration/addons.md:89]: This sentence is removed.

File: content/zh-cn/docs/concepts/policy/resource-quotas.md (hunk 1 above)
Comment (Sea-n) [inline content/zh-cn/docs/concepts/policy/resource-quotas.md:55]: I think "enforced" in this context is just "執行", not including "強制".

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 3]
```diff
@@ -193,19 +193,7 @@ client_address=10.240.0.3
...
-  node2-. SNAT .->node1[节点 1];
-  node1-. SNAT .->node2;
-  node1-->endpoint(端点);
-
-  classDef plain fill:#ddd,stroke:#fff,stroke-width:4px,color:#000;
-  classDef k8s fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
-  class node1,node2,endpoint k8s;
-  class client plain;
-{{</ mermaid >}}
+{{< figure src="/docs/images/tutor-service-nodePort-fig01.svg" alt="source IP nodeport figure 01" class="diagram-large" caption="Figure. Source IP Type=NodePort using SNAT" link="https://mermaid.live/edit#pako:eNqNkV9rwyAUxb-K3LysYEqS_WFYKAzat9GHdW9zDxKvi9RoMIZtlH732ZjSbE970cu5v3s86hFqJxEYfHjRNeT5ZcUtIbXRaMNN2hZ5vrYRqt52cSXV-4iMSuwkZiYtyX739EqWaahMQ-V1qPxDVLNOvkYrO6fj2dupWMR2iiT6foOKdEZoS5Q2hmVSStoH7w7IMqXUVOefWoaG3XVftHbGeZYVRbH6ZXJ47CeL2-qhxvt_ucTe1SUlpuMN6CX12XeGpLdJiaMMFFr0rdAyvvfxjHEIDbbIgcVSohKDCRy4PUV06KQIuJU6OA9MCdMjBTEEt_-2NbDgB7xAGy3i97VJPP0ABRmcqg" >}}
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:196]: This image use `/docs/images` page instead of `/zh-cn/docs/images`, maybe we don't have to duplicate the svg file.

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 4]
```diff
@@ -246,19 +234,7 @@ client_address=104.132.1.79
...
-  node1 --> endpoint(端点);
-  endpoint --> node1;
-
-  classDef plain fill:#ddd,stroke:#fff,stroke-width:4px,color:#000;
-  classDef k8s fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
-  class node1,node2,endpoint k8s;
-  class client plain;
-{{</ mermaid >}}
-
+{{< figure src="/docs/images/tutor-service-nodePort-fig02.svg" alt="source IP nodeport figure 02" class="diagram-large" caption="Figure. Source IP Type=NodePort preserves client source IP address" link="" >}}
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:237]: Figure caption and alternative text can be translated.

File: content/zh-cn/docs/concepts/policy/resource-quotas.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/policy/resource-quotas.md:55]: I don't agree.

File: content/zh-cn/docs/concepts/cluster-administration/addons.md (hunk 2 above)
Comment (howieyuen) [reply content/zh-cn/docs/concepts/cluster-administration/addons.md:89]: no, here it is, https://github.com/kubernetes/website/blob/cc08154e70de230720cc793caf706a3612b58919/content/en/docs/concepts/cluster-administration/addons.md?plain=1#L33

Comment (howieyuen) [reply content/zh-cn/docs/concepts/cluster-administration/addons.md:89]: line 89 is removed...

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 5]
```diff
@@ -240,25 +227,12 @@ client_address=104.132.1.79
 
 * 客户端发送数据包到 `node2:nodePort`，它没有任何 endpoints
 * 数据包被丢弃
-* 客户端发送数据包到 `node1:nodePort`，它*有*endpoints
+* 客户端发送数据包到 `node1:nodePort`，它*有* endpoints
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:230]: In Mandarin, we use **bold** instead of *italic*.
```suggestion
* 客户端发送数据包到 `node1:nodePort`，它**有** endpoints
```

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 6]
```diff
@@ -240,25 +227,12 @@ client_address=104.132.1.79
...
-  node1 --> endpoint(端点);
-  endpoint --> node1;
-
-  classDef plain fill:#ddd,stroke:#fff,stroke-width:4px,color:#000;
-  classDef k8s fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
-  class node1,node2,endpoint k8s;
-  class client plain;
-{{</ mermaid >}}
-
+{{< figure src="/docs/images/tutor-service-nodePort-fig02.svg" alt="图 2：源 IP NodePort" class="diagram-large" caption="Type=NodePort，源 IP 保存客户端源 IP 地址" link="" >}}
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:235]: The caption is displayed same as normal text, maybe add `圖 N` will be better.
```suggestion
{{< figure src="/docs/images/tutor-service-nodePort-fig02.svg" alt="图 2：源 IP NodePort" class="diagram-large" caption="圖 2：Type=NodePort，源 IP 保存客户端源 IP 地址" link="" >}}
```

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 7]
```diff
@@ -240,25 +227,12 @@ client_address=104.132.1.79
 
 * 客户端发送数据包到 `node2:nodePort`，它没有任何 endpoints
 * 数据包被丢弃
-* 客户端发送数据包到 `node1:nodePort`，它*有*endpoints
+* 客户端发送数据包到 `node1:nodePort`，它*有* endpoints
 * node1 使用正确的源 IP 地址将数据包路由到 endpoint
 
 
 用图表示：
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:234]: This whole file have no `<!-- original content -->`, could you add them back?
It will be huge in diff result, maybe we can split to another PR.

File: content/zh-cn/docs/images/tutor-service-nodePort-fig01.svg [hunk 8]
```diff
@@ -0,0 +1 @@
...m="translate(0,0)" class="label"><g transform="translate(-24.828125,-12)"><foreignObject height="24" width="49.65625"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">Node 2</div></foreignObject></g></g></g><g style="opacity: 1;" transform="translate(308.578125,42)" id="flowchart-node1-15" class="node k8s"><rect class="label-container" height="44" width="69.65625" y="-22" x="-34.828125" ry="0" rx="0"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-24.828125,-12)"><foreignObject height="24" width="49.65625"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">Node 1</div></foreignObject></g></g></g><g style="opacity: 1;" transform="translate(435.09375,42)" id="flowchart-endpoint-19" class="node k8s"><rect class="label-container" height="44" width="83.375" y="-22" x="-41.6875" ry="5" rx="5"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-31.6875,-12)"><foreignObject height="24" width="63.375"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">Endpoint</div></foreignObject></g></g></g></g></g></g></svg>
```
Comment (Sea-n) [inline content/zh-cn/docs/images/tutor-service-nodePort-fig01.svg:1]: This two svg file are not in use, maybe can be removed.

File: content/zh-cn/docs/tutorials/services/source-ip.md (hunk 7 above)
Comment (Sea-n) [reply content/zh-cn/docs/tutorials/services/source-ip.md:234]: Not sure about the priority.
@tengqm Do you think it should be done now or someday after the umbrella issue closed?

File: content/zh-cn/docs/tutorials/services/source-ip.md (hunk 6 above)
Comment (tengqm) [reply content/zh-cn/docs/tutorials/services/source-ip.md:235]: we may want to reference the copied file rather than the one under 'content/en/'.

Comment (howieyuen) [reply content/zh-cn/docs/tutorials/services/source-ip.md:235]: I will add back these two images, and change url prefix to 'zh-cn'

File: content/zh-cn/docs/tutorials/services/source-ip.md (hunk 7 above)
Comment (howieyuen) [reply content/zh-cn/docs/tutorials/services/source-ip.md:390]: Thanks for noticing. I retranslate this file and add original text

File: content/zh-cn/docs/images/tutor-service-nodePort-fig01.svg (hunk 8 above)
Comment (howieyuen) [reply content/zh-cn/docs/images/tutor-service-nodePort-fig01.svg:1]: just as tengqm said, we should reference copied file instead of "en/docs/image"

**Comment by howieyuen** [comment]
Thanks to @Sea-n , notice there is no original text in this file(`content/zh-cn/docs/tutorials/services/source-ip.md`),
so I retranslate, and it is out-of-sync for a long long time.
OMG, it costs me over 3 hours to complete...

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 9]
```diff
@@ -1,122 +1,206 @@
...
+: replacing the destination IP on a packet; in this page, that usually means replacing with the IP address of a {{< glossary_tooltip term_id="pod" >}}
 
+[VIP](/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies)
+: a virtual IP address, such as the one assigned to every {{< glossary_tooltip text="Service" term_id="service" >}} in Kubernetes
 
-本文使用了下列术语：
+[kube-proxy](/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies)
+: a network daemon that orchestrates Service VIP management on every node
+-->
+[NAT](https://en.wikipedia.org/wiki/Network_address_translation)
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:58]: Change it to `[[網絡地址轉換]]`.
```suggestion
[NAT](https://zh.wikipedia.org/wiki/%E7%BD%91%E7%BB%9C%E5%9C%B0%E5%9D%80%E8%BD%AC%E6%8D%A2)
```

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 10]
```diff
@@ -134,10 +218,12 @@ ip addr
        valid_lft forever preferred_lft forever
 ```
 
-然后使用 `wget` 去请求本地 Web 服务器
+<!--
+…then use `wget` to query the local webserver
+-->
+……然后使用 `wget` 查询本地网络服务器：
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:224]:
```suggestion
……然后使用 `wget` 查询本地網頁服务器：
```

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 11]
```diff
@@ -147,259 +233,413 @@ command=GET
...
+[preserve the client source IP](/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip).
+If you set `service.spec.externalTrafficPolicy` to the value `Local`,
+kube-proxy only proxies proxy requests to local endpoints, and does not
+forward traffic to other nodes. This approach preserves the original
+source IP address. If there are no local endpoints, packets sent to the
+node are dropped, so you can rely on the correct source-ip in any packet
+processing rules you might apply a packet that make it through to the
+endpoint.
+-->
+为避免这种情况，Kubernetes 有一个特性可以 [保留客户端源 IP](/zh-cn/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip)。
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:328]: Should be either remove the space or break into 2 lines.
```suggestion
为避免这种情况，Kubernetes 有一个特性可以[保留客户端源 IP](/zh-cn/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip)。
```

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 12]
```diff
@@ -147,259 +233,413 @@ command=GET
...
+headers, or the
+[proxy protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt).
+Load balancers in the second category can leverage the feature described above
+by creating an HTTP health check pointing at the port stored in
+the `service.spec.healthCheckNodePort` field on the Service.
+-->
+第一类负载均衡器必须使用负载均衡器和后端之间商定的协议来传达真实的客户端 IP，
+例如 HTTP [转发](https://tools.ietf.org/html/rfc7239#section-5.2)或
+[X-FORWARDED-FOR](https://en.wikipedia.org/wiki/X-Forwarded-For)
+头部，或[代理协议](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt)。
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:614]: Not sure how we translate `HTTP header`.
```suggestion
表頭，或[代理协议](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt)。
```

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 13]
```diff
@@ -1,122 +1,206 @@
...
 clusterip    ClusterIP   10.0.170.92   <none>        80/TCP    51s
 ```
 
-
-从相同集群中的一个 pod 访问这个 `ClusterIP`：
-
+<!-- 
+And hitting the `ClusterIP` from a pod in the same cluster:
+-->
+并从同一集群中的 Pod 中点击 `ClusterIP`：
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:186]: Not sure should we translate `hitting` as `訪問` or `點擊`.

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 14]
```diff
@@ -147,259 +233,413 @@ command=GET
...
 ```
 
-打印Service的IPs：
+<!-- 
+Print out the IP addresses of the Service:
+-->
+打印 Service 的 IP 地址：
 ```console
 kubectl get svc loadbalancer
 ```
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:433]:
````suggestion
打印 Service 的 IP 地址：
```shell
kubectl get svc loadbalancer
```
````

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 15]
```diff
@@ -147,259 +233,413 @@ command=GET
...
+kube-proxy only proxies proxy requests to local endpoints, and does not
+forward traffic to other nodes. This approach preserves the original
+source IP address. If there are no local endpoints, packets sent to the
+node are dropped, so you can rely on the correct source-ip in any packet
+processing rules you might apply a packet that make it through to the
+endpoint.
+-->
+为避免这种情况，Kubernetes 有一个特性可以 [保留客户端源 IP](/zh-cn/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip)。
+如果将 `service.spec.externalTrafficPolicy` 设置为 `Local`，
+kube-proxy 只会将代理请求代理到本地端点，而不会将流量转发到其他节点。
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:330]:
```suggestion
kube-proxy 只会将代理请求代理到本地端点，而不会将流量转发到其他节点。
```

**Review by Sea-n — COMMENTED** [review]
Thanks for the hard work! 🙇‍♂️

Preview: https://deploy-preview-34237--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tutorials/services/source-ip/

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 16]
```diff
@@ -147,259 +233,413 @@ command=GET
...
+```yaml
   healthCheckNodePort: 32122
 ```
 
+<!-- 
+The `service.spec.healthCheckNodePort` field points to a port on every node
+serving the health check at `/healthz`. You can test this:
+-->
+`service.spec.healthCheckNodePort` 字段指向每个节点在
+`/healthz` 路径上用于健康检查的端口。你可以这样测试：
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/services/source-ip.md:509]:
```suggestion
`service.spec.healthCheckNodePort` 字段指向每个在 `/healthz`
路径上提供健康检查的节点的端口。你可以这样测试：
```

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 17]
```diff
@@ -147,259 +233,413 @@ command=GET
...
+## Cross-platform support
+
+Only some cloud providers offer support for source IP preservation through
+Services with `Type=LoadBalancer`.
+The cloud provider you're running on might fulfill the request for a loadbalancer
+in a few different ways:
+-->
+## 跨平台支持  {#cross-platform-support}
+
+只有部分云提供商通过 `Type=LoadBalancer` 的 Service 提供保存源 IP 的支持。
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/services/source-ip.md:582]:
```suggestion
只有部分云提供商为 `Type=LoadBalancer` 的 Service 提供保存源 IP 的支持。
```

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 18]
```diff
@@ -1,122 +1,206 @@
...
 
-* [NAT](https://en.wikipedia.org/wiki/Network_address_translation): 网络地址转换
-* [Source NAT](https://en.wikipedia.org/wiki/Network_address_translation#SNAT): 替换数据包的源 IP, 通常为节点的 IP
-* [Destination NAT](https://en.wikipedia.org/wiki/Network_address_translation#DNAT): 替换数据包的目的 IP, 通常为 Pod 的 IP
-* [VIP](/zh/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies): 一个虚拟 IP, 例如分配给每个 Kubernetes Service 的 IP
-* [Kube-proxy](/zh/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies): 一个网络守护程序，在每个节点上协调 Service VIP 管理
+[Source NAT](https://en.wikipedia.org/wiki/Network_address_translation#SNAT)
+: 替换数据包上的源 IP；在此页面中，这通常意味着替换为节点的 IP 地址
 
+[Destination NAT](https://en.wikipedia.org/wiki/Network_address_translation#DNAT)
```
[the remark this answers is not in the document]
Comment (Sea-n) [reply content/zh-cn/docs/tutorials/services/source-ip.md:64]: There have no SNAT or DNAT section in zh/NAT page.

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 19]
```diff
@@ -1,122 +1,206 @@
...
+## 先决条件  {#prerequisites}
 
-```console
+{{< include "task-tutorial-prereqs.md" >}}
+
+<!-- 
+The examples use a small nginx webserver that echoes back the source
+IP of requests it receives through an HTTP header. You can create it as follows:
+-->
+这些示例使用一个小型 nginx 网络服务器，它通过 HTTP 标头返回它接收到的请求的源 IP。
```
[the remark this answers is not in the document]
Comment (Sea-n) [reply content/zh-cn/docs/tutorials/services/source-ip.md:84]: If we use upper case `Web`, should it be `NginX` or `nginx`?

File: content/zh-cn/docs/concepts/policy/resource-quotas.md [hunk 20]
```diff
@@ -53,8 +52,7 @@ Resource quotas work like this:
   the `LimitRanger` admission controller to force defaults for pods that make no compute resource requirements.
   See the [walkthrough](/docs/tasks/administer-cluster/quota-memory-cpu-namespace/) for an example of how to avoid this problem.
 -->
-- 不同的团队可以在不同的命名空间下工作，目前这是非约束性的，在未来的版本中可能会通过
-  ACL (Access Control List 访问控制列表) 来实现强制性约束。
+- 不同的团队可以在不同的命名空间下工作。这可以通过 [RBAC](/zh/docs/reference/access-authn-authz/rbac/) 强制执行。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/policy/resource-quotas.md:55]: 后续路径都要改成 `zh-cn`了吧，例如：
/zh-cn/docs/reference/access-authn-authz/rbac/

File: content/zh-cn/docs/concepts/cluster-administration/addons.md [hunk 21]
```diff
@@ -84,9 +84,8 @@ Add-ons 扩展了 Kubernetes 的功能。
   CaaS / PaaS 平台（例如关键容器服务（PKS）和 OpenShift）之间的集成。
 * [Nuage](https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.1.1-1/docs/kubernetes-1-installation.rst)
   是一个 SDN 平台，可在 Kubernetes Pods 和非 Kubernetes 环境之间提供基于策略的联网，并具有可视化和安全监控。
-* Romana 是一个 pod 网络的第三层解决方案，并支持
-  [NetworkPolicy API](/zh/docs/concepts/services-networking/network-policies/)。
-  Kubeadm add-on 安装细节可以在[这里](https://github.com/romana/romana/tree/master/containerize)找到。
+* [Romana](https://github.com/romana) 是一个 Pod 网络的第三层解决方案，并支持
+  [NetworkPolicy](/zh/docs/concepts/services-networking/network-policies/) API。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/cluster-administration/addons.md:88]: /zh-cn/docs/concepts/services-networking/network-policies/

File: content/zh-cn/docs/concepts/policy/resource-quotas.md (hunk 20 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/policy/resource-quotas.md:55]: 暂时无所谓，顺便改掉也好。
待 34221 完全关闭，我们做一次批量修改。

File: content/zh-cn/docs/tutorials/services/source-ip.md (hunk 9 above)
Comment (howieyuen) [reply content/zh-cn/docs/tutorials/services/source-ip.md:58]: I tried to visit zh page as well, and I found this page doesn't have all content. so I leave them unchanged.

[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh-cn/docs/tutorials/services/source-ip.md:58]: 此页是有的，但是下面2个就没有，也就是中文翻译不全，所以我就没改过来

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 22]
```diff
@@ -147,259 +233,413 @@ command=GET
...
+are source NAT'd by default. You can test this by creating a `NodePort` Service:
+-->
+## `Type=NodePort` 类型 Service 的源 IP  {#source-ip-for-services-with-type-nodeport}
 
-
-从 Kubernetes 1.5 开始，发送给类型为 [Type=NodePort](/zh/docs/user-guide/services/#nodeport) Services 的数据包默认进行源地址 NAT。你可以通过创建一个 `NodePort` Service 来进行测试：
-
-```console
+默认情况下，使用 [`Type=NodePort`](/zh-cn/docs/concepts/services-networking/service/#type-nodeport)
+发送到 Service 的数据包经过源 NAT。你可以通过创建一个 `NodePort` 的 Service 来测试它：
```
[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh-cn/docs/tutorials/services/source-ip.md:251]: I think "f发送" is a typo 😆

File: content/zh-cn/docs/tutorials/services/source-ip.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh-cn/docs/tutorials/services/source-ip.md]: https://zh.wikipedia.org/wiki/%E7%BD%91%E7%BB%9C%E5%9C%B0%E5%9D%80%E8%BD%AC%E6%8D%A2

[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh-cn/docs/tutorials/services/source-ip.md]: 这个我已经改过了，Sea-n 也给了我这个链接

**Comment by tengqm** [comment]
My approval stands.

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 23]
```diff
@@ -1,122 +1,206 @@
...
-* [NAT](https://en.wikipedia.org/wiki/Network_address_translation): 网络地址转换
-* [Source NAT](https://en.wikipedia.org/wiki/Network_address_translation#SNAT): 替换数据包的源 IP, 通常为节点的 IP
-* [Destination NAT](https://en.wikipedia.org/wiki/Network_address_translation#DNAT): 替换数据包的目的 IP, 通常为 Pod 的 IP
-* [VIP](/zh/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies): 一个虚拟 IP, 例如分配给每个 Kubernetes Service 的 IP
-* [Kube-proxy](/zh/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies): 一个网络守护程序，在每个节点上协调 Service VIP 管理
+[Source NAT](https://en.wikipedia.org/wiki/Network_address_translation#SNAT)
+: 替换数据包上的源 IP；在本页面中，这通常意味着替换为节点的 IP 地址
 
+[Destination NAT](https://en.wikipedia.org/wiki/Network_address_translation#DNAT)
+：替换数据包上的目标 IP；在本页面中，这通常意味着替换为 {{<glossary_tooltip text="Pod" term_id="pod" >}} 的 IP 地址
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/services/source-ip.md:65]: The format is broken, please refer to [preview page](https://deploy-preview-34237--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tutorials/services/source-ip/#terminology).
```suggestion
: 替换数据包上的目标 IP；在本页面中，这通常意味着替换为 {{<glossary_tooltip text="Pod" term_id="pod" >}} 的 IP 地址
```

**Review by Sea-n — COMMENTED** [review]
My last suggestion, other parts are all looks good to me!

File: content/zh-cn/docs/tutorials/services/source-ip.md (hunk 23 above)
Comment (howieyuen) [reply content/zh-cn/docs/tutorials/services/source-ip.md:65]: thanks.

**Comment by Sea-n** [comment]
Thanks!
/lgtm
